src/data/contributors/tanya21034.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please, delete this file.

Reply:
Also delete sample.txt that you have created.

Reply:
Please check and suggest if any other changes are to be made. Thanks!